tests/e2e-test.spec.js

```diff
@@ -93,27 +94,26 @@ describe('Main workflow Test Suite', () => {
     const pgTableContact = await client.query(`SELECT * from ${PGTABLE} where _id = $1`, [contact._id]);
     expect(pgTableContact.rows[0]._deleted).to.equal(true);
     await delay(6); // wait for DBT
-    const modelContactResult = await client.query(`SELECT * FROM ${pgSchema}.contacts where _id = $1`, [contact._id]);
-    expect(modelContactResult.rows.length).to.equal(1);
-    expect(modelContactResult.rows[0]._deleted).to.equal(true);
+    const modelContactResult = await client.query(`SELECT * FROM ${pgSchema}.contacts where uuid = $1`, [contact._id]);
+    expect(modelContactResult.rows.length).to.equal(0);
```

Comment:
What would happen if the contact deleted was of the type ´person´?

Reply:
We would expect the contact to be deleted from both the `contact` and `person` tables.